AntennaTracker/wscript

```diff
@@ -5,8 +5,8 @@ def build(bld):
     vehicle = bld.path.name
     bld.ap_stlib(
         name=vehicle + '_libs',
-        vehicle=vehicle,
-        libraries=bld.ap_common_vehicle_libraries() + [
+        ap_vehicle=vehicle,
+        ap_libraries=bld.ap_common_vehicle_libraries() + [
             'AC_PID',
         ],
         use='mavlink',
```

Comment:
As talked in the previous PR there's no need for having this use mavlink because we are always enforcing it now